usaspending_api/etl/es_transaction_template.json

```diff
@@ -570,7 +570,15 @@
       },
       "disaster_emergency_fund_codes": {
           "type": "keyword"
+        },
```

Comment:
Obviously won't block the PR on this, but if you wouldn't mind correcting the spacing of the `}` and line above it that would be amazing 🙏